stale_repos.py

```diff
@@ -95,10 +95,11 @@ def get_inactive_repos(github_connection, inactive_days_threshold, organization)
         if last_push_str is None:
             continue
         last_push = parse(last_push_str)
+        last_push_disp_date = parse(last_push_str).date().isoformat()
```

Comment:
```suggestion
        last_push_disp_date = last_push.date().isoformat()
```
Seems best to only parse it once.

Reply:
Oh I missed that. Thanks